src/lib/utils/metadata.ts

```diff
@@ -1,7 +1,18 @@
 import type { AuthorData, PostsData } from "$routes/blog/content";
 
 export const DEFAULT_HOST = 'https://appwrite.io';
-export const DEFAULT_DESCRIPTION = 'Appwrite is an open-source platform for building applications at any scale, using your preferred programming languages and tools.';
+export const DEFAULT_DESCRIPTION = 'Appwrite is an open-source backend platform offering essential APIs for building web and mobile applications.';
+export const ASSETS_DESCRIPTION = 'Resources for presenting the Appwrite brand, ensuring consistency in using our logos, colours, and other brand elements across various platforms and materials.'
```

Comment:
we dont need consts for descriptions that are only used once, feel free to just move them to the places where they are used 👍🏻

Reply:
Done.